src/infra/metrics.rs

```diff
@@ -66,7 +66,7 @@ metrics! {
     histogram storage_read_logs{success},
 
     "Duration of storage read_slot operation."
-    histogram storage_read_slot{point_in_time, success},
+    histogram storage_read_slot{point_in_time, success, origin},
```

Comment:
keep success always as the last parameter as all the other metrics
origin can be the first one because it is a category